README.md

```diff
@@ -1,164 +1,111 @@
 
 ![](doc/source/images/architecture.png)
 
 ## Flow
 
-* Log into IBM's DSX service.
-
-* Upload the data as a data asset into DSX.
-
-* Start a notebook in DSX and input the data asset previously created.
-
-* Follow following discussion and steps.
-
-1. Introduction and Background.
+1. Log into IBM's Watson Studio service.
+2. Upload the data as a data asset into Watson Studio.
+3. Start a notebook in Watson Studio and input the data asset previously created.
+4. Train machine learning models with scikit learn.
+5. Use XGBoost to run the classification.
+6. Evaluate their prediction performance.
 
-2. Data Set Description.
-
-3. Statement of Classification Problem.
-
-4. Software and Tools(Xgboost and Scikit Learn).
-
-5. Visual Data Exploration to understand data (Using seaborn and matplotlib).
-
-6. Create Scikit learn ML Pipelines for Data Processing.
+## Included components
 
-7. Model Training.
+* [IBM Watson Studio](https://dataplatform.ibm.com): Analyze data using RStudio, Jupyter, and Python in a configured, collaborative environment that includes IBM value-adds, such as managed Spark.
+* [Jupyter Notebook](http://jupyter.org/): An open source web application that allows you to create and share documents that contain live code, equations, visualizations, and explanatory text.
 
-  7.1 What and Why of XGBoost.
+## Featured technologies
 
-  7.2 Discuss Metrics for Model Performance.
+* [Data Science](https://medium.com/ibm-data-science-experience/): Systems and scientific methods to analyze structured and unstructured data in order to extract knowledge and insights.
+* [Python](https://www.python.org/): Python is a programming language that lets you work more quickly and integrate your systems more effectively.
+* [XGBoost](https://github.com/dmlc/xgboost): Extreme Gradient Boosting is decision tree based tools for creating ML model.
+* [Scikit Learn](http://scikit-learn.org/stable/):  A Python library for providing efficient tools for data mining and machine learning.
+* [Pandas](http://pandas.pydata.org/): A Python library providing high-performance, easy-to-use data structures.
+* [Matplotlib](https://matplotlib.org/): A Python library integrating matplot for visualization.
+* [SeaBorn](https://seaborn.pydata.org/): Another higher level Python library for visualization.
 
-  7.3 First Attempt at Model Training and it's performance,evaluation and analysis.
+# Steps
 
-  7.4 Strategy For Better Classifier for the Imbalance Data
+This Code Pattern consists of following activities:
 
-  7.5 Second Attempt at Model Training using Weighted Samples and it's performance, evaluation and analysis.
+* [Run a Jupyter notebook in the IBM Watson Studio](#run-using-a-jupyter-notebook-in-the-ibm-watson-studio).
```

Comment:
you could add a step here about data being loaded in, not sure if we put it in an object store or not, but it's something that happens.

Reply:
Yes, we need clarification on that from Alok.